docs/src/restart.md

```diff
@@ -0,0 +1,85 @@
+# [Restart Simulation](@id restart)
+
+You can continue running an already finished simulation by first
+preparing the simulation for the restart and then performing the restart.
+Here we suppose that in the first run your simulation stops at time 1.0
+and then you want it to run further to time 2.0.
+
+## [Prepare the simulation for a restart](@id restart_preparation)
+In you original elixir you need to specify to write out restart files.
+Those will later be read for the restart of your simulation.
+This is done almost the same way as writing the snapshots using the
+```SaveSolutionCallback``` callback.
+For the restart files it is called ```SaveRestartCallback```:
+```julia
+save_restart = SaveRestartCallback(interval=100,
+                                   save_final_restart=true)
+```
+Make this part of your ```CallbackSet```.
```

Comment:
```suggestion
Make this part of your `CallbackSet`.
```
Single backticks are enough for inline code. Could you please update the other parts as well?